packages/crypto/src/jose.ts

```diff
@@ -863,7 +863,7 @@ export class Jose {
     const jose = multicodecToJoseMapping[lookupKey];
 
     if (jose === undefined) {
-      throw new Error(`Unsupported Multicodec to JOSE conversion: '${options.name ?? options.code}'`);
```

Comment:
this is unreachable;
we can't hit this line with undefined code AND name, as undefined code would throw in `getNameFromCode` with undefined name

Removed to reach 100% tests coverage, as it's not possible to cover unreachable edge case...

Reply:
Appreciate the clarification